backend/btrixcloud/orgs.py

```diff
@@ -817,6 +828,19 @@ async def update_quotas(
 
         return {"updated": True}
 
+    @router.post("/read-only", tags=["organizations"])
+    async def update_read_only(
+        update: OrgReadOnlyUpdate,
+        org: Organization = Depends(org_owner_dep),
```

Comment:
Does this allow an arbitrary org to be specified? I'm unsure how `Depends` works here

Reply:
This particular router requires `/org/<id>` in the path, so the org id will be specified already.

Reply:
The `Depends` here ensures that the person calling the endpoint is at least an owner of the org, though can possibly remove that since we're checking for superadmin as part of the endpoint.

Reply:
Ah, right, makes sense!